ios/FluentUI.Demo/FluentUI.Demo/Demos/IndeterminateProgressBarDemoController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class IndeterminateProgressBarDemoController: DemoController {
+	override func viewDidLoad() {
```

Comment:
make sure you follow the contribution guidelin and install swiftlint . Also, we use 4 spaces instead of tab to index. please make sure to change your xcode preference setting and update all of your files